app/javascript/components/region-form/index.jsx

```diff
@@ -0,0 +1,67 @@
+import React, { useEffect, useState } from 'react';
+import PropTypes from 'prop-types';
+import { Grid } from 'patternfly-react';
+import createSchema from './region-form.schema';
+import MiqFormRenderer from '../../forms/data-driven-form';
+import { API } from '../../http_api';
+import miqRedirectBack from "../../helpers/miq-redirect-back";
+
+const RegionForm = ({ id, maxDescLen }) => {
+  const [description, setDescription] = useState({});
```

Comment:
The description should be `undefined` and not an empty object when mounting the component:
```suggestion
  const [description, setDescription] = useState();
```